strings_lab.js

```diff
@@ -0,0 +1,73 @@
+// 1. `console.log` the first character of a string.
```

Comment:
Overall good understanding of string methods and properties. Try to solve bonus questions in future assignments as they are good practice. Keep practicing, keep coding and keep up the good work!